Core/AeroGear.Mobile/IServiceModule.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+namespace Core.AeroGear.Mobile
+{
+    public interface IServiceModule
```

Comment:
Any objections of using I prefix for interfaces? It's quite common in .NET world. I actually like to use personally in Java too and especially in Kotlin.